### DISCUSSION TITLE
Unable to login to my Vercel account as it was created using an email that no longer exitsts

### DISCUSSION THREAD

**Discussion by swarup-hobspace** [vercel/community#7240] [Account Issue] [answered]
### Question

I have my project deployed on a different account that I no longer have email access to since it was deleted. I would like to switch the ownership to this account. I am in possesion of both the github accounts.

**Comment by amyegan** [comment, marked as answer]
Hey @swarup-hobspace. For security reasons, I cannot safely discuss account details here in public. Please send a message to [registration@vercel.com](mailto:registration@vercel.com) with the GitHub usernames and emails associated with your Vercel accounts. 

[vercel.com/guides/how-to-get-vercel-support](https://vercel.com/guides/how-to-get-vercel-support)

[a comment by an automated account is not shown]
